app/src/main/java/com/duckduckgo/app/di/AndroidBindingModule.kt

```diff
@@ -177,6 +178,9 @@ abstract class AndroidBindingModule {
     @ContributesAndroidInjector
     abstract fun brokenSiteNegativeFeedbackFragment(): BrokenSiteNegativeFeedbackFragment
 
+    @ContributesAndroidInjector
+    abstract fun siteLocationPermissionDialog(): SiteLocationPermissionDialog
```

Comment:
curious why location permission dialog change in this PR?

Reply:
We need to inject `FaviconManager` to show the favicon in the dialog.